src/parser/vast_parser.js

```diff
@@ -231,30 +231,19 @@ export class VASTParser extends EventEmitter {
    * Returns a Promise which resolves with the array or rejects with an error according to the result of the parsing.
    * @param  {Object} vastXml - An object representing an xml document.
    * @param  {Object} options - An optional Object of parameters to be used in the parsing process.
-   * @emits  VASTParser#VAST-resolving
-   * @emits  VASTParser#VAST-resolved
-   * @return {Promise}
+   * @return {Array}
    */
-  parse(
-    vastXml,
-    {
-      resolveAll = true,
-      wrapperSequence = null,
-      originalUrl = null,
-      wrapperDepth = 0,
-      isRootVAST = false
-    }
-  ) {
+  parseVastXml(vastXml, { isRootVAST = false }) {
```

Comment:
Just 2 more things:
- Please add a `@throws` doc about the error too (see http://usejsdoc.org/tags-throws.html)
- The description should be changed to something like "Parses the given xml Object into an array of ads, throws an error in case an invalid VAST XML is provided"

Reply:
👍 done!
I hadn't even removed the description part saying "Returns promise".... shame on me. Good catch! :)